spyder_kernels/console/kernel.py

```diff
@@ -653,6 +734,37 @@ def _set_mpl_backend(self, backend, pylab=False):
 
         self._mpl_backend_error = error
 
+    def _set_config_option(self, option, value):
+        """
+        Set config options using the %config magic.
+
+        As parameters:
+            option: config option, for example 'InlineBackend.figure_format'.
+            value: value of the option, for example 'SVG', 'Retina', etc.
+        """
+        from IPython.core.getipython import get_ipython
+        try:
+            base_config = "{option} = "
+            value_line = (
+                "'{value}'" if isinstance(value, TEXT_TYPES) else "{value}")
+            config_line = base_config + value_line
+            get_ipython().run_line_magic(
+                'config',
+                config_line.format(option=option, value=value))
+        except Exception:
+            pass
+
+    def _set_mpl_inline_rc_config(self, option, value):
+        """
+        Update any of the Matplolib rcParams given an option and value.
+        """
+        try:
+            from matplotlib import rcParams
+            rcParams[option] = value
+        except ImportError:
```

Comment:
This should be `except Exception` to avoid other possible errors when importing Matplotlib.